### PR TITLE
Updated Search

### PR DESCRIPTION
**Pull request by jblss888**
I updated the lookUpRecord() and made it more more versatile.
There is a new method that is called searchRecords() that takes it two inputs, fieldType and itemtoSearch. It then uses a switch case to change what is being put into the MySQL query and will print out the query. This reduces the need for repetitive code to print out the result tabel

### CODE REVIEW

File: mySqlDatabaseRemote/src/mySqlDriver2/inventory_db_remote.java [hunk 1]
```diff
@@ -9,19 +9,17 @@
...
 import java.util.Scanner;
 
+
 public class inventory_db_remote {
 
-	public static String url = "";
-	public static String username = "";
-	public static String password = "";
+	public static String url = "jdbc:mysql://192.254.233.63:3306/fbacon_spilledcoffee_main_dev";
+	public static String username = "fbacon_team_3250";
```
Comment (Naomimmm) [inline mySqlDatabaseRemote/src/mySqlDriver2/inventory_db_remote.java:16]: Did you mean to check these in?

Comment (Naomimmm) [inline mySqlDatabaseRemote/src/mySqlDriver2/inventory_db_remote.java:16]: Did you mean to check these in?

**Review by Naomimmm — DISMISSED** [review]
just left a minor comment about the credentials, otherwise looks good!

**Review by ssok305 — APPROVED** [review]

File: mySqlDatabaseRemote/src/mySqlDriver2/inventory_db_remote.java (hunk 1 above)
Comment (jblss888) [reply mySqlDatabaseRemote/src/mySqlDriver2/inventory_db_remote.java:16]: thank you!

**Review by ssok305 — APPROVED** [review]

**Comment by jblss888** [comment]
No longer relevent

**Pull request closed without merging**